docs/Advanced-Topics-Entities.md

```diff
@@ -4,7 +4,7 @@ title: Entities
 layout: docs
 category: Advanced Topics
 next: v0-10-api-migration
-permalink: docs/advanced-topics-entities.html
+permalink: docs/Advanced-Topics-Entities.md
```

Comment:
I don't think these links are actually broken after checking them on the live site: https://draftjs.org/docs/advanced-topics-entities.html. These markdown files are converted into html with the correct names/paths as shown here.

Reply:
All good @carlesandres! Have a great day.